TestResultSummaryService/perf/DataManagerAggregate.js

```diff
@@ -22,40 +21,48 @@ class DataManagerAggregate {
                 }
             }
         }
-        return {name, variant, jdkBuildDate, benchmarkMetricsCollection};
+        return {name, variant, benchmarkMetricsCollection};
     }
 
-    static async updateBuildWithAggregateInfo(id, testResultsDB, name, variant, jdkBuildDate, metricsCollection) {
+    static async updateBuildWithAggregateInfo(hasChildren, id, testResultsDB, benchmarkName, benchmarkVariant, jdkDate, javaVersion, metricsCollection) {
         // calculate the aggregate data
-        if (name != null && variant != null && jdkBuildDate != null && metricsCollection != null) {
+        let validAggregateInfo = false;
```

Comment:
Please add comment here as requested before to specify the requirements for `validAggregateInfo`.